### DISCUSSION TITLE
en/docs/05

### DISCUSSION THREAD

**Discussion by giscus** [yuorb/yuorb.github.io#13] [General]
# en/docs/05

5.0 Verb Morphology Formatives functioning as verbs in a sentence, rather than nouns, may display several additional morphological categories not found in nouns, including Valen...

https://yuorb.github.io/en/docs/05.html

**Comment by snow-lumi** [comment]
The "is comprised of" is arguably proscribed usage, but it's more common and imo easier to read than the prescribed construction. Reference: https://en.wiktionary.org/wiki/comprise verb 2nd meaning.
I don't think it needs to be "corrected".

**Reply by ryanlo713** [reply]
It is advisable to substitute the verb with "consists of."
